"Check for CWE-798 in codeql-eg-HardcodedCredentials-b","","warning","cwe-798: Compare at line 20 is a password comparison with a string literal","/CWE-306_MAC-1a.py","20","41","20","64"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-b","","warning","cwe-798: Compare at line 23 is a password comparison with a string literal","/CWE-306_MAC-2a.py","23","36","23","54"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-b","","warning","cwe-798: Compare at line 27 is a password comparison with a string literal","/CWE-306_MAC-2a.py","27","37","27","54"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-434_UUF-1a.py","14","5","14","15"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-434_UUF-1a.py","15","5","15","23"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-434_UUF-1b.py","14","5","14","15"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-434_UUF-1b.py","15","5","15","23"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-434_UUF-2a.py","14","46","14","56"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-434_UUF-2a.py","15","46","15","64"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-434_UUF-2b.py","17","9","17","27"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-434_UUF-2b.py","18","9","18","31"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-502_DUD-2b.py","19","5","19","23"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-502_DUD-2b.py","20","5","20","27"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-502_DUD-2c.py","23","9","23","27"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-502_DUD-2c.py","24","9","24","31"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-522_IPC-3c.py","11","5","11","23"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-522_IPC-3c.py","12","5","12","27"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-89_SQI-2c.py","13","5","13","17"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-89_SQI-2c.py","14","5","14","25"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-89_SQI-3a.py","13","5","13","17"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-89_SQI-3a.py","14","5","14","25"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-89_SQI-3b.py","13","44","13","54"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-89_SQI-3b.py","13","57","13","75"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-89_SQI-3b.py","44","44","44","54"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-89_SQI-3b.py","44","57","44","75"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-89_SQI-3c.py","12","26","12","40"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-89_SQI-3c.py","13","26","13","44"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","user","/CWE-89_SQI-3c.py","28","26","28","40"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-c","","warning","password","/CWE-89_SQI-3c.py","29","26","29","44"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-a","","warning","cwe-798: Compare at line 20 is a password comparison with a string literal","/CWE-306_MAC-1a.py","20","41","20","64"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-a","","warning","cwe-798: Compare at line 23 is a password comparison with a string literal","/CWE-306_MAC-2a.py","23","36","23","54"
"Check for CWE-798 in codeql-eg-HardcodedCredentials-a","","warning","cwe-798: Compare at line 27 is a password comparison with a string literal","/CWE-306_MAC-2a.py","27","37","27","54"
"Reflected server-side cross-site scripting","Writing user input directly to a web page allows for a cross-site scripting vulnerability.","error","Cross-site scripting vulnerability due to [[""a user-provided value""|""relative:///CWE-434_UUF-3b.py:65:19:65:26""]].","/CWE-434_UUF-3b.py","66","12","78","24"
"Reflected server-side cross-site scripting","Writing user input directly to a web page allows for a cross-site scripting vulnerability.","error","Cross-site scripting vulnerability due to [[""a user-provided value""|""relative:///CWE-502_DUD-1a.py:5:26:5:32""]].","/CWE-502_DUD-1a.py","42","12","42","15"
"Reflected server-side cross-site scripting","Writing user input directly to a web page allows for a cross-site scripting vulnerability.","error","Cross-site scripting vulnerability due to [[""a user-provided value""|""relative:///CWE-502_DUD-1c.py:5:26:5:32""]].","/CWE-502_DUD-1c.py","21","16","21","32"
"Clear-text storage of sensitive information","Sensitive information stored without encryption or hashing can expose it to an attacker.","error","[[""Sensitive data (secret)""|""relative:///CWE-732_IPA-3b.py:8:15:8:49""]] is stored here.","/CWE-732_IPA-3b.py","12","13","12","23"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-200_ESI-2a.py","58","5","58","23"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-200_ESI-3b.py","21","5","21","23"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-20_IIV-2c.py","22","5","22","23"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-22_ILP-2b.py","19","5","19","23"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-306_MAC-1a.py","37","5","37","23"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-306_MAC-3a.py","52","5","52","23"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-434_UUF-1c.py","30","5","30","23"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-434_UUF-3a.py","47","5","47","23"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-434_UUF-3b.py","82","5","82","23"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-522_IPC-2c.py","30","5","30","34"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-89_SQI-3b.py","61","5","61","23"
"Flask app is run in debug mode","Running a Flask app in debug mode may allow an attacker to run arbitrary code through the Werkzeug debugger.","error","A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger.","/CWE-89_SQI-3c.py","50","5","50","23"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///CWE-502_DUD-1c.py:22:30:22:30""]] flows to this location and may be exposed to an external user.","/CWE-502_DUD-1c.py","23","16","23","28"
"Hard-coded credentials","Credentials are hard coded in the source code of the application.","error","Use of [[""hardcoded credentials""|""relative:///CWE-306_MAC-3a.py:12:24:12:42""]].","/CWE-306_MAC-3a.py","23","34","23","60"
"Hard-coded credentials","Credentials are hard coded in the source code of the application.","error","Use of [[""hardcoded credentials""|""relative:///CWE-306_MAC-3c.py:12:24:12:42""]].
Use of [[""hardcoded credentials""|""relative:///CWE-306_MAC-3c.py:13:24:13:42""]].","/CWE-306_MAC-3c.py","23","34","23","60"
"Uncontrolled command line","Using externally controlled strings in a command line may allow a malicious user to change the meaning of the command.","error","This command line depends on [[""a user-provided value""|""relative:///CWE-78_INS-2b.py:5:26:5:32""]].","/CWE-78_INS-2b.py","17","42","17","44"
"Uncontrolled command line","Using externally controlled strings in a command line may allow a malicious user to change the meaning of the command.","error","This command line depends on [[""a user-provided value""|""relative:///CWE-78_INS-2c.py:12:10:12:12""]].","/CWE-78_INS-2c.py","23","42","23","56"
"Polynomial regular expression used on uncontrolled data","A regular expression that can require polynomial time to match may be vulnerable to denial-of-service attacks.","warning","This [[""regular expression""|""relative:///CWE-20_IIV-1b.py:11:31:11:32""]] that depends on [[""a user-provided value""|""relative:///CWE-20_IIV-1b.py:6:36:6:42""]] may run slow on strings with many repetitions of 'a'.","/CWE-20_IIV-1b.py","18","38","18","43"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on [[""a user-provided value""|""relative:///CWE-22_ILP-2a.py:11:9:11:12""]].","/CWE-22_ILP-2a.py","20","27","20","34"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on [[""a user-provided value""|""relative:///CWE-22_ILP-2a.py:11:9:11:12""]].","/CWE-22_ILP-2a.py","27","22","27","29"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on [[""a user-provided value""|""relative:///CWE-22_ILP-2b.py:11:9:11:16""]].","/CWE-22_ILP-2b.py","14","27","14","39"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on [[""a user-provided value""|""relative:///CWE-22_ILP-2b.py:11:9:11:16""]].","/CWE-22_ILP-2b.py","16","22","16","34"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on [[""a user-provided value""|""relative:///CWE-434_UUF-1a.py:7:26:7:32""]].","/CWE-434_UUF-1a.py","40","19","40","26"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on [[""a user-provided value""|""relative:///CWE-434_UUF-1b.py:7:26:7:32""]].","/CWE-434_UUF-1b.py","40","19","40","26"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on [[""a user-provided value""|""relative:///CWE-434_UUF-1c.py:6:26:6:32""]].","/CWE-434_UUF-1c.py","25","15","25","22"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on [[""a user-provided value""|""relative:///CWE-434_UUF-3b.py:6:26:6:32""]].","/CWE-434_UUF-3b.py","47","19","47","69"
